src/core.js

```diff
@@ -746,7 +746,7 @@ define([
             return false;
         });
 
-        $modalBody.find('#findUsagesSearch').on('keyup inserted.atwho', function () {
+        $modalBody.find('#findUsagesSearch').on('keyup keypress inserted.atwho', function (e) {
```

Comment:
Might as well remove the `keyup` rather than firing two events for each character.